readabilipy/__main__.py

```diff
@@ -65,8 +73,7 @@ def main():
         use_readability=(not args.use_python_parser),
     )
 
-    with open(args.output_file, "w", encoding="utf-8") as j:
-        json.dump(article, j, ensure_ascii=False)
+    json.dump(article, output_file, ensure_ascii=False)
```

Comment:
Similarly, if `output_file` *is* a file, it should be closed here.

Reply:
Done.